crates/runtime/tests/iceberg/mod.rs

```diff
@@ -1,194 +1,2 @@
-                engine: Some("duckdb".to_string()),
-                mode: Mode::File,
-                ..Default::default()
-            });
-
-            let rt = run_iceberg_test(
-                "iceberg_dataset_test",
-                dataset.clone(),
-                "SELECT * FROM customer LIMIT 10",
-                false,
-                None,
-            )
-            .await?;
-
-            drop(rt);
-
-            let _ = run_iceberg_test(
-                "iceberg_dataset_test",
-                dataset,
-                "SELECT * FROM customer LIMIT 10",
-                true,
-                Some("iceberg_integration_test_duckdb_acceleration_restart"),
-            )
-            .await?;
-
-            Ok(())
-        })
-        .await
-}
-
-async fn run_iceberg_test(
-    app_name: &str,
-    dataset: Dataset,
-    query: &str,
-    assert_snapshot: bool,
-    snapshot_name: Option<&str>,
-) -> Result<Runtime, anyhow::Error> {
-    let app = AppBuilder::new(app_name).with_dataset(dataset).build();
-
-    let status = status::RuntimeStatus::new();
-    let df = get_test_datafusion(Arc::clone(&status));
-
-    let rt = Runtime::builder()
-        .with_app(app)
-        .with_datafusion(df)
-        .with_runtime_status(status)
-        .build()
-        .await;
-
-    tokio::select! {
-        () = tokio::time::sleep(std::time::Duration::from_secs(30)) => {
-            panic!("Timeout waiting for components to load");
-        }
-        () = rt.load_components() => {}
-    }
-
-    runtime_ready_check(&rt).await;
-
-    let mut result = rt.datafusion().query_builder(query).build().run().await?;
-
-    let mut results: Vec<RecordBatch> = vec![];
-    while let Some(batch) = result.data.next().await {
-        results.push(batch?);
-    }
-
-    if assert_snapshot {
-        let pretty = arrow::util::pretty::pretty_format_batches(&results)
-            .map_err(|e| anyhow::Error::msg(e.to_string()))?;
-
-        insta::assert_snapshot!(snapshot_name.unwrap_or_default(), pretty);
-    }
-
-    Ok(rt)
-}
-
-fn make_iceberg_dataset(
-    namespace: &str,
-    table: &str,
-    name: &str,
-) -> Result<Dataset, anyhow::Error> {
-    let account_id =
-        std::env::var("AWS_ICEBERG_ACCOUNT_ID").context("AWS_ICEBERG_ACCOUNT_ID is not set")?;
-    let region = std::env::var("AWS_ICEBERG_REGION").context("AWS_ICEBERG_REGION is not set")?;
-    let _ = std::env::var("AWS_ACCESS_KEY_ID").context("AWS_ACCESS_KEY_ID is not set")?;
-    let _ = std::env::var("AWS_SECRET_ACCESS_KEY").context("AWS_SECRET_ACCESS_KEY is not set")?;
-
-    let from = format!("iceberg:https://glue.ap-northeast-2.amazonaws.com/iceberg/v1/catalogs/{account_id}/namespaces/{namespace}/tables/{table}");
-    let mut dataset = Dataset::new(from, name);
-    dataset.params = Some(DatasetParams::from_string_map(HashMap::from([(
-        "iceberg_s3_region".to_string(),
-        region,
-    )])));
-    Ok(dataset)
-}
+mod iceberg_catalog;
```

Comment:
nit: you could have just called these `catalog` and `connector`, having the `iceberg_` prefix is redundant. Can come in a follow-up PR.